assignments/prototypes.js

```diff
@@ -14,34 +12,79 @@
   * name
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
+  cons
 */
+const GameObject = function(att){
+  this.createdAt = att.createdAt;
+  this.name = att.name;
+  this.dimensions = att.dimensions;
+}
+GameObject.prototype.destroy = function(){
+  console.log   (`${this.name} was removed from the game.`)
+}
 
+const Orc = new GameObject ({
+  createdAt: "April2",
+  name:"Ihab",
+  dimensions:"8 feet"
+})
+console.log(Orc.name)
+  Orc.destroy()
 /*
   === CharacterStats ===
   * healthPoints
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
+ function CharacterStats(stats){
+   GameObject.call(this, stats);
+   this.healthPoints = stats.healthPoints;
+     this.takeDamage = function() {
+   console.log(`${this.name} took damage.`)
+ }
```

Comment:
since you have this method defined in lines 49-51, probably, you do not need this function here ;)